health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FormulaConfiguration.js

```
@@ -1,319 +1,291 @@
+                label={t("YES")}
+                onButtonClick={() => {
+                  handleConfirmDelete();
+                }}
+              />,
+              <Button type={"button"} size={"large"} variation={"primary"} label={t("NO")} onButtonClick={handleCancelDelete} />,
+            ]}
+            sortFooterChildren={true}
+            onClose={() => {
+              setShowPopUp(false);
+            }}
+          ></PopUp>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)